app/src/test/java/ch/hikemate/app/model/route/saved/SavedHikesRepositoryFirebaseTest.kt

```diff
@@ -0,0 +1,63 @@
+package ch.hikemate.app.model.route.saved
+
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+
+class SavedHikesRepositoryFirebaseTest {
```

Comment:
Same comment as before, this shouldn't be a RepositoryFirebaseTest, since there is no Firebase notion inside, even if this is still WIP

Reply:
Done in [f371674c](https://github.com/HikeMate/hikeMateApp/pull/66/commits/f371674cd55b28a97502642354fc44a7b7071f43).